packages/orchestrator-ui-components/src/components/WfoForms/formFields/SubscriptionSummaryField.tsx

```diff
@@ -33,12 +32,10 @@ interface SubscriptionSummaryDisplayProps {
 const SubscriptionSummaryDisplay = ({
     subscriptionId,
 }: SubscriptionSummaryDisplayProps) => {
-    const { data } = useQueryWithGraphql(
-        GET_SUBSCRIPTION_DETAIL_GRAPHQL_QUERY,
-        { subscriptionId },
-        `subscription-${subscriptionId}`,
-    );
-    const subscriptionDetail = data?.subscriptions.page[0];
+    const { data } = useGetSubscriptionDetailQuery({
+        subscriptionId,
+    });
+    const subscriptionDetail = data?.subscriptions[0];
```

Comment:
Isn't this always going to be 1 result. I would prefer moving this destucturing to the endpoint so you data is the subscriptionDetail right away